examples/water/dpa3/run.sh

```
@@ -0,0 +1,23 @@
+# unset PADDLE_ELASTIC_JOB_ID
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)